src/luarocks/build/rust-mlua.lua

```diff
@@ -35,6 +35,10 @@ function mlua.run(rockspec, no_install)
     end
 
     local cmd = {"cargo build --release"}
+
+    local target_path = rockspec.build and rockspec.build.target_path or "target"
+    table.insert(cmd, "--target=" .. target_path)
```

Comment:
it could be problematic if `target_path` has whitespaces or other non-escaped characters

Reply:
How about with `fs.Q()`?

Reply:
Thanks!